utils/cams_wind_pipeline.py

```
@@ -0,0 +1,265 @@
+import logging
+import tempfile
+import shutil
+
+# === Logging Setup ===
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+class CamsDownload:
```

Pylint (3.3.7)

[refactor] Too many instance attributes (8/7)

(R0902)